lib/wrappingInteger.fz

```diff
@@ -188,3 +197,5 @@ wrappingIntegers(W (wrappingInteger W).type) : numerics W is
 
   min W is abstract
   max W is abstract
+
+  all_bits_set W is abstract
```

Comment:
please add a comment describing what this returns. Also, copy this comment to all implementations of this. The comment will end up in the API documentation on https://flang.dev/docs/, which looks a bit sad without any documentation. 
Same as with snake_case: We need comments for existing features as well, but in separate PRs.